ArjunaJTA/jta/pom.xml

```diff
@@ -21,6 +21,9 @@
     <!-- java.net.preferIPv6Addresses=false is a workaround to (not) solve the bug in arquillian-extension-byteman -->
     <ipv6.server.jvm.args>-Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1]
       -Djboss.bind.address.unsecure=[::1] -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=false</ipv6.server.jvm.args>
+    <byteman.agent.support>-Djboss.modules.system.pkgs=com.sun.tools.attach,org.jboss.byteman -Djdk.attach.allowAttachSelf=true</byteman.agent.support>
+        <!-- required for VM attach under JDK 11-->
```

Comment:
Could you please align this?

Reply:
Thanks, updated.

Reply:
I don't think you pushed the change with the alignment corrected, @marcosgopen? I am going to go ahead and approve it already and if all you change is the alignment here then feel free to consider my approval standing, otherwise please can you re-request a review from me?

Reply:
Sorry, my bad. Now it should be there. Thanks